examples/panel-basic/README.md

```diff
@@ -1,16 +1,16 @@
-# Grafana Basic Panel Plugin example
+# Basic Panel Plugin for Grafana
 
-This example provides a template for how to build a basic panel plugin. The definition for the panel is provided in `/src/components/SimplePanel/SimplePanel.tsx`.
+This repository contains an example plugin for Grafana called Basic Panel Plugin. It serves as a foundational example for creating custom panel plugins within Grafana.
```

Comment:
```suggestion
Basic Panel Plugin is a foundational example for creating custom panel plugins within Grafana.
```

"This repository" feels odd when the user is currently in a subfolder? Similar concern for the rest of the overviews